datamining/procedures/get_lending_stats.py

```diff
@@ -0,0 +1,26 @@
+from django.db import connection
+
+
+def get_lending_stats(start_date, end_date) -> dict[str, int]:
```

Comment:
I think this bit can use a little comment to what we are dragging together and what we are getting out of that because it takes a bit to long to read it from the code

Reply:
done